### ISSUE TITLE
[feature]: Find issues that can be worked on based on blockedBy relation and status of blocking issue

### ISSUE THREAD

**Issue by glassp** [makeplane/plane#3137]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Summary

There should be a filter for issues that are currently unblocked (relation: blockedBy & those related issue in status `DONE`)
Alternatively it would be fine to have some new status `BLOCKED` that gets automatically set to `TODO` if all blocking issues are `DONE` or `CANCELED`

### Why should this be worked on?

Currently it is not easy to find issues that can be worked on without clicking on all issues and checked if they are blocked by another issue and then check if the blocking issues are done or not

**Comment by vihar** [comment]
@glassp, this is a great feature request. We will shortly begin working on a PRD and share it with the community. 

In the meantime, if you don't mind sharing, could you provide some insight into the use case that prompted you to make this feature request? Perhaps providing an example could assist us in better understanding your needs.

**Comment by glassp** [comment]
### Feature request reason

Im working on a project X but I have dependencies on some helper libraries that have their own seperate project
I need to know on which tickets I can work in Project X, even if I don't have access to the helper lib project.
 
Like when a project manager with access to both projects sets the issue on project x as blocked by an issue on the helper library it is in needs to be visible to anyone with access to the issue on Project X that the issue is blocked and if the blocking has been resolved (e.g. blocking issue is done)


### Wished mechanics


There are 3 Projects A, B, C
With Tickets A-1 ... A-5, B-1... B5 and C-1 ... C5 and the developer for a given project (which does not have access to the other projects) wants to find the tickets that can be worked on. 

Lets assume they want to work on Project A but 
  - `A-1` is blocked by B-1 which is `in Progress`, 
  - `A-2` is blocked by B-2 which is blocked by `C-2`, and `C-2` is done but `B-2` is still `open`
  - `A-3` is blocked by B-3 which is `done`
  - `A-4` is the only `open` ticket that is not blocked by anything
  - `A-5` is `in progress`
  
It should be possible to create a fiter or a view that shows all tickets in project A that can be actively worked on (`open` & not blocked)
So in this example I'd want a view that shows
 - `A-3`
 - `A-4`
 
Since they are in `TODO` and their blocking tickets are done or they were not blocked

### Nice to have extra feature
 
If a list of all tickets that block others is feasible that would be nice. But having a list of possible workload is more important, especially when working with many projects that have dependencies on each other

**Comment by ari-party** [comment]
Been a while! Bump?

**Comment by joshmcorreia** [comment]
Based on [the docs](https://docs.plane.so/core-concepts/issues/visualise_filter) they use "Is blocking is True" as a use case for Boolean properties, but this doesn't seem to actually be a filter in the UI from what I can see...
